File: Leetcode Challenge/April/Java/Container With Most Water.java
class Solution {
    public int maxArea(int[] height) {
        int ans=0;
        int left=0;
        int right=height.length-1;
        while (left<=right){
            ans=Math.max(Math.min(height[left],height[right])*(right-left),ans);
            if(height[left]<height[right]){
                left++;
            }else{
                right--;
            }
        }
        return ans;
    }
}

File: Leetcode Challenge/April/Python/Container With Most Water.py
class Solution:
    def maxArea(self, height: List[int]) -> int:
        ans=0;
        left=0;
        right=len(height)-1;
        while (left<=right):
            ans=max(min(height[left],height[right])*(right-left),ans)
            if height[left]<height[right]:
                left+=1
            else:
                right-=1
            
        return ans
